packages/plugins/typescript/react-apollo/tests/__snapshots__/react-apollo.spec.ts.snap

```diff
@@ -1,5 +1,48 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
+exports[`React Apollo Apollo Client Sdk generation should generate output correctly 1`] = `
+Object {
+  "content": "
+export const FeedDocument = gql\`
+    query Feed {
+  feed {
+    id
+    commentCount
+    repository {
+      full_name
+      html_url
+      owner {
+        avatar_url
+      }
+    }
+  }
+}
+    \`;
+export type FeedQueryResult = Apollo.QueryResult<FeedQuery, FeedQueryVariables>;
+export const SubmitRepositoryDocument = gql\`
+    mutation SubmitRepository($name: String) {
+  submitRepository(repoFullName: $name) {
+    id
+  }
+}
+    \`;
+export type SubmitRepositoryMutationResult = Apollo.MutationResult<SubmitRepositoryMutation>;
+export type SubmitRepositoryMutationOptions = Apollo.BaseMutationOptions<SubmitRepositoryMutation, SubmitRepositoryMutationVariables>;
+export const getSdk = (client: ApolloClient<any>) => {
+      FeedQuery(variables?: FeedQueryVariables) {
+          return client.query<FeedQuery>({variables, query: FeedDocument)
+      },
+SubmitRepositoryMutation(variables?: SubmitRepositoryMutationVariables) {
+          return client.query<SubmitRepositoryMutation>({variables, query: SubmitRepositoryDocument)
+      }
+    };",
+  "prepend": Array [
+    "import { gql } from '@apollo/client';",
+    "import * as Apollo from '@apollo/client';",
+  ],
+}
```

Comment:
because the prepend has only Apollo, I think I'll need to change the type of client from: `ApolloClient<any>` to `Apollo.ApolloClient<any>`